src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -30,14 +33,20 @@ internal sealed class ManagedProjectSystemPackage : AsyncPackage
 
         protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
+            await TaskScheduler.Default;
+
             IComponentModel componentModel = await this.GetServiceAsync<SComponentModel, IComponentModel>();
             
             IEnumerable<IPackageService> packageServices = componentModel.GetExtensions<IPackageService>();
 
+            var sw = Stopwatch.StartNew();
+
             foreach (IPackageService packageService in packageServices)
             {
                 await packageService.InitializeAsync(this);
             }
+
+            Debug.WriteLine($"Initialized managed language project system package services in {sw.Elapsed.TotalMilliseconds:,0.##} ms");
```

Comment:
Would the TraceUtilities class be useful for logging this statistic on release builds?

Reply:
Possibly. I pushed a WIP commit to continue working on this at @davkean's office yesterday (where we all met) so this change will likely be removed. Unfortunately GitHub doesn't allow converting a PR to a draft (only the other way around).